Protocol.md

```diff
@@ -318,16 +318,14 @@ Frame Contents
     +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
     |                           Stream ID                           |
     +-----------+-+-+---------------+-------------------------------+
-    |Frame Type |0|M|      Flags    |
+    |Frame Type |0|0|      Flags    |
     +-----------+-+-+---------------+-------------------------------+
     |                          Error Code                           |
     +---------------------------------------------------------------+
-                        Metadata & Error Data
+                               Error Data
 ```
 
 * __Frame Type__: (6 bits = max value 63) 0x0B
-* __Flags__: (10 bits)
-     * (__M__)etadata: Metadata present
 * __Error Code__: (32 bits = max value 2^31-1 = 2,147,483,647) Type of Error.
      * See list of valid Error Codes below.
 * __Error Data__: includes Payload describing error information. Error Data SHOULD be a UTF-8 encoded string. The string MUST NOT be null terminated.
```

Comment:
My reading of this is that Error data does not use either Metadata or Data mimetype.  Error data is always just a UTF-8 string.

Is that correct?